content/hi/docs/tutorials/kubernetes-basics/_index.html

```diff
@@ -0,0 +1,117 @@
+---
+title: बुनियादी कुबेरनेट्स सीखें
+linkTitle: बुनियादी कुबेरनेट्स सीखें
+no_list: true
+weight: 10
+card:
+  name: tutorials
+  weight: 20
+  title: मूल बातें
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+  <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+  <main class="content">
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>बुनियादी कुबेरनेट्स</h2>
+        <p>यह ट्यूटोरियल कुबेरनेट्स क्लस्टर ऑर्केस्ट्रेशन सिस्टम की मूल बातों का पूर्वाभ्यास प्रदान करता है। प्रत्येक मॉड्यूल में कुबेरनेट्स की प्रमुख विशेषताओं और अवधारणाओं पर कुछ पृष्ठभूमि की जानकारी होती है, और इसमें एक इंटरैक्टिव ऑनलाइन ट्यूटोरियल शामिल होता है। ये इंटरेक्टिव ट्यूटोरियल आपको अपने लिए एक साधारण क्लस्टर और इसके कंटेनरीकृत अनुप्रयोगों का प्रबंधन करने देते हैं।</p>
+        <p>इंटरैक्टिव ट्यूटोरियल का उपयोग करके, आप सीख सकते हैं:</p>
+        <ul>
+        <li>एक क्लस्टर पर एक कंटेनरीकृत अनुप्रयोग डिप्लॉय करें।</li>
+        <li>डिप्लॉयमेंट को स्केल करें।</li>
+        <li>एक नए सॉफ़्टवेयर संस्करण के साथ कंटेनरीकृत एप्लिकेशन को अपडेट करें।</li>
+        <li>कंटेनरीकृत एप्लिकेशन को डीबग करें।</li>
+        </ul>
+        <p>ट्यूटोरियल आपके वेब ब्राउज़र में एक वर्चुअल टर्मिनल चलाने के लिए कटाकोडा का उपयोग करते हैं जो मिनिक्यूब चलाता है, कुबेरनेट्स की एक छोटे पैमाने पर स्थानीय डिप्लॉयमेंट जो कहीं भी चल सकती है। कोई सॉफ़्टवेयर स्थापित करने या कुछ भी कॉन्फ़िगर करने की कोई आवश्यकता नहीं है; प्रत्येक इंटरैक्टिव ट्यूटोरियल सीधे आपके वेब ब्राउज़र से ही चलता है।</p>
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>कुबेरनेट्स आपके लिए क्या कर सकता है?</h2>
+        <p>आधुनिक वेब सेवाओं के साथ, उपयोगकर्ता उम्मीद करते हैं कि एप्लिकेशन 24/7 उपलब्ध होंगे, और डेवलपर्स उन एप्लिकेशन के नए संस्करणों को दिन में कई बार तैनात करने की उम्मीद करते हैं। कंटेनरीकरण पैकेज सॉफ़्टवेयर को इन लक्ष्यों को पूरा करने में मदद करता है, जिससे एप्लिकेशन बिना डाउनटाइम के रिलीज़ और अपडेट किए जा सकते हैं। Kubernetes आपको यह सुनिश्चित करने में मदद करता है कि वे कंटेनरीकृत एप्लिकेशन जहां और जब आप चाहते हैं, चलते हैं, और उन्हें उन संसाधनों और उपकरणों को खोजने में मदद करते हैं जिनकी उन्हें काम करने की आवश्यकता होती है। कुबेरनेट्स एक प्रोडक्शन-रेडी, ओपन सोर्स प्लेटफॉर्म है जिसे कंटेनर ऑर्केस्ट्रेशन में गूगल के संचित अनुभव के साथ डिज़ाइन किया गया है, जो समुदाय के सर्वश्रेष्ठ-नस्लीय विचारों के साथ संयुक्त है।</p>
```

Comment:
nit: Would a Hindi reader confidently understand “24/7”? Are there some readers who would and others who might not?

Reply:
I think everyone will easily understand 24/7. It is possible to write numbers in devnagri but in real life, they are almost extinct and everyone just uses Latin 1, 2, 3 ,4 ...
24/7 is a common term used in India too.